source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -45,9 +45,17 @@ def delete_journeys(responses, request):
     if request.get('debug', False):
         return
 
+    keep_jo = request.get("_jo", False)
+
+    def to_delete(journey):
```

Comment:
I recommend you take https://github.com/hove-io/navitia/blob/fe7c69a69dc4cb6331d98adf920f94da3d72da96/source/jormungandr/jormungandr/scenarios/journey_filter.py#L164 as an example how a filter is handled... it seems to be more tedious, in practice,  all filters are gathered in only one place... and it would be easier for the user to debug...

Reply:
Maybe I don't get your comment, but I want some journeys **not** to be filtered by any filter. 

For example, on this [request](https://playground.navitia.io/play.html?request=https%3A%2F%2Fhormungandr.corefront.sbx.aws.private%2Fv1%2Fcoverage%2Fidfm%2Fjourneys%3Ffrom%3Dstop_area%253AIDFM%253A71607%26to%3Dpoi%253A14945%26datetime%3D20230512T121903%26_pt_planner%3Dloki%26debug%3Dtrue%26criteria%3Darrival_stop_attractivity%26_walking_transfer_penalty%3D360%26) the second journey is filtered because it is similar to the first, but I want to keep it because it ends at a stop that have a high capacity.
Note that we also want to keep the first journey.